plaso/output/shared_elastic.py

```diff
@@ -360,6 +364,18 @@ def Close(self):
 
     self._client = None
 
+  def SetAdditionalFields(self, additional_fields):
+    """Set the additional field.
+
+    Args:
+      additional_fields (dict): A dict with additional fields and values that
```

Comment:
remove `A dict with`, change `dict` to `dict[str, object]`

why do the values need to be passed here and not just the field names ?

Reply:
it absolutely needs to be a key/value pair, otherwise this would not be needed at all...

Reply:
I understand that it is still a draft. I'm also trying to understand the changes to prevent having them to change a lot later on.

Could you give me some context on why both key and value are needed?

Reply:
done

and why do values, because that's the goal here, to add a key/value pair to each event that is ingested, to make timeline filtering work in TS.

explained later